bin/webpack-dev-server.js

```diff
@@ -331,9 +331,12 @@ function processOptions(wpOpt) {
 
 	if(argv["open"] || argv["open-page"]) {
 		options.open = true;
-		options.openPage = argv["open-page"] || "";
```

Comment:
Maybe I'm having an off day, but I don't see how this is any different..

```js
var a = {};
a['b'] || ''  // <== ""
```

Is `argv["open-page"]` returning `"undefined"` as a String here for some reason?